packages/components/src/guide/index.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useMediaQuery } from '@wordpress/compose';
+import { useState, Children } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Modal from '../modal';
+import KeyboardShortcuts from '../keyboard-shortcuts';
+import IconButton from '../icon-button';
+import PageControl from './page-control';
+import { BackButtonIcon, ForwardButtonIcon } from './icons';
+import FinishButton from './finish-button';
+
+export default function Guide( { children, className, finishButtonText, onFinish } ) {
+	const isMobile = useMediaQuery( '(max-width: 600px)' );
```

Comment:
Would be good to abstract the threshods in a `useViewport` hook like we do in the viewports package but this is probably a separate PR I think.

Reply:
The was `isMobile` is used in this component though makes me wonder if it should be just a CSS media query :)

Reply:
I had thought that using CSS here didn't play nicely with the auto-focus behaviour that `FinishButton` has, but it seems okay now! Switched to CSS in 9b5e1dd17.

(We should still deprecate `@wordpress/viewport` though—it's a good idea.)